src/iceflow/itrf/converter.py

```diff
@@ -34,9 +32,8 @@ def sinceEpoch(date):
     return date.year + fraction
 
 
-@pa.check_types()
 def transform_itrf(
-    data: DataFrame[commonDataColumns],
+    data: IceFlowData,
```

Comment:
`data` here is typed with the parent class. Normally we'd expect dataset-specific subclasses to be passed in.

Reply:
Can it be any subclass, or only specific ones? I.e. should we use union instead?